pymongo/collection.py

```diff
@@ -1483,6 +1483,10 @@ def find_raw_batches(self, *args, **kwargs):
         .. note:: find_raw_batches does not support sessions or auto
            encryption.
 
+        .. versionchanged:: 3.12
+           Instead of ignoring the user-specified read concern, this method
+           now sends it to the server.
```

Comment:
"when connected to MongoDB 3.6+"?

Reply:
Its actually 3.2+ AFAICT. changed.

Reply:
Can you explain? Judging from the test failures it's 3.6+: https://evergreen.mongodb.com/build/mongo_python_driver_tests_storage_engines__platform~awslinux_python_version~3.6_storage_engine~mmapv1_c36ec37281be1d663f8eba1169b29b09fffef451_21_06_07_19_12_49

Reply:
Never mind, you are right... Its only when OP_MSG is available.